yellow/client/advanced/sculpt.py

```diff
@@ -148,19 +150,41 @@ def check_asset_status(self, uuid: str) -> Dict:
         status_data = json.loads(response.content.decode())
         return status_data     
     
-    def fetch_asset(self, uuid: str, output_dir: str) -> str:
+    def fetch_asset(
+            self, 
+            uuid: str, 
+            output_dir: str, 
+            file_format: str = FileFormatEnum.OBJ,
+            rig_type: str = RigTypeEnum.NO_RIG,
+        )-> str:
         """Fetch/download an generated asset.
 
         Args:
             uuid (str): UUID of an asset
             output_dir (str): Directory to store an asset
+            file_format (str): File format of an asset
+            rig_type (str): Rig type applied to a mesh
 
         Raises:
             ValueError: Error recevied from the Yellow API during fetching an asset
 
         Returns:
             str: Output path
         """
+        try:
+            file_format = FileFormatEnum(file_format)
```

Comment:
nit: I think that the stdlib `ValueError` that's returned is already self explanatory enough here and we could avoid this additional `try/except` overhead from these blocks, wdyt?

```
>>> @enum.unique
... class Test(enum.StrEnum):
...   FOO = enum.auto()
...   BAR = enum.auto()
... 
>>> Test('bats')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/ctew/.pyenv/versions/3.11.9/lib/python3.11/enum.py", line 714, in __call__
    return cls.__new__(cls, value)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ctew/.pyenv/versions/3.11.9/lib/python3.11/enum.py", line 1137, in __new__
    raise ve_exc
ValueError: 'bats' is not a valid Test
```

Reply:
True. A user will only lose immediate information about what formats or types are possible to use, but can easily check it in the code

Reply:
Done